coreapi/main_api.py

```diff
@@ -197,8 +197,8 @@ def facerecogniseinimage(request, filename, network):
 
     logger.info(msg="facerecogniseinimage called")
     file_path = os.path.join(MEDIA_ROOT, 'images/' + filename)
-    handle_uploaded_file(request.FILES['file'], file_path)
-    file = request.FILES['file']
+    handle_uploaded_file(input_file, file_path)
+    file = input_file
```

Comment:
unnecessary file variable.

Reply:
removed. similarly for other places as well